vin/test/controllers/vin/subs_controller_test.rb

```diff
@@ -1,19 +0,0 @@
-require 'test_helper'
-
-class Vin::SubsControllerTest < ActionController::TestCase
-  test "should get create" do
```

Comment:
yep, i have to implement full unit testing, so that's the next step (even before implementing the remaining use cases)